### PR TITLE
Css flexbox

### PR DESCRIPTION
**Pull request by vkahlon**
Screen Shot 2021-12-02 at 11 09 09 AM
Screen Shot 2021-12-02 at 11 06 49 AM
Screen Shot 2021-12-02 at 11 08 01 AM
Screen Shot 2021-12-02 at 11 08 37 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Vick! You seem to have accidentally committed your Brochure Form code on this branch. I can help you get this cleaned up later on after you finish the current exercise 👍

File: css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        flex-basis: calc(100% / 6);
```
Comment (codingcodymiller) [inline css-flexbox/flex-wrap.html:15]: `flex-basis` is a property meant to be applied to the child elements of a flexbox, not the flexbox itself. This should be being applied to the `.card-wrapper` class

File: css-flexbox/justify-content.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```
Comment (codingcodymiller) [inline css-flexbox/justify-content.html:14]: `flex-basis` is a property meant to be applied to the child elements of a flexbox, not the flexbox itself. This should be being applied to the `.card-wrapper` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-flexbox/justify-content.html (hunk 2 above)
Comment (vkahlon) [reply css-flexbox/justify-content.html]: Made the necessary changes.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**